### PR TITLE
fix: make inkwells visible

### PR DESCRIPTION
**Pull request by 55nknown**
- added onTap to CustomCard

## Examples:
image
image
image
image

### CODE REVIEW

**Comment by ponces** [comment]
Could you please resolve conflicts so we can merge this later?

**Comment by 55nknown** [comment]
Done.

File: lib/ui/widgets/appInfoView/app_info_view.dart [hunk 1]
```diff
@@ -61,135 +61,123 @@ class AppInfoView extends StatelessWidget {
...
-                                          Icons
-                                              .settings_backup_restore_outlined,
+                                          Icons.build_outlined,
                                           color: Theme.of(context)
                                               .colorScheme
                                               .primary,
                                         ),
                                         const SizedBox(height: 10),
                                         I18nText(
-                                          'appInfoView.unpatchButton',
```
Comment (ponces) [inline lib/ui/widgets/appInfoView/app_info_view.dart:192]: Why was the unpatch button removed altogether?

**Review by ponces — CHANGES_REQUESTED** [review]

File: lib/ui/widgets/shared/custom_card.dart [hunk 2]
```diff
@@ -3,30 +3,33 @@ import 'package:flutter/material.dart';
...
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      decoration: BoxDecoration(
+    return Material(
+      type: isFilled ? MaterialType.card : MaterialType.transparency,
+      color: isFilled
+          ? Theme.of(context).colorScheme.secondaryContainer.withOpacity(.4)
```
[the remark this answers is not in the document]
Comment (55nknown) [reply lib/ui/widgets/shared/custom_card.dart:22]: Sure

**Comment by ponces** [comment]
Thanks!

**Pull request merged**